source/funkin/data/stage/StageData.hx

```diff
@@ -202,6 +202,16 @@ typedef StageDataCharacter =
   @:default([0, 0])
   var position:Array<Float>;
 
+  /**
+   * If set to true, when positioning the character, the position origin will be at vertical bottom and horizontal center of the sprite.
+   * If set to false, when positioning the character, the position origin will be at the top-left of the sprite.
+   * This prevents characters from not being properly positioned.
+   * @default true
+   */
+  @:optional
+  @:default(true)
+  var fromOrigin:Bool;
```

Comment:
I feel like this is a poor name choice but I can't think of a better one off the top of my head (maybe something to do with the fact that this is changing the sprite's anchor point?)

Reply:
maybe `cornerAnchor`/`useCornerAnchor`? also I will flip the conditions if this name is used